src/Landing/redux/actions/LatestArticles.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-console */
+import axios from 'axios';
+import { FETCH_LATEST_POSTS } from '../types';
+
+const fetchLatestArticles = () => (dispatch) => {
+  axios.get('https://ah-backend-realers-staging.herokuapp.com/api/articles/?format=json&limit=9')
+    .then(res => dispatch({
+      type: FETCH_LATEST_POSTS,
+      payload: res.data.results,
+    }))
+    .catch((error) => {
+      console.error('Error fetching articles', error);
```

Comment:
Clearly this is not how you want to handle the error that comes back. You probably want to display something to the user in the application.

Reply:
Hello handling has been implemented as you may see below

<img width="804" alt="Screenshot 2019-07-16 at 17 18 24" src="https://user-images.githubusercontent.com/21230051/61302033-cb375680-a7ed-11e9-9cde-30c6335f6bc4.png">